x/auth/client/testutil/suite.go

```diff
@@ -426,6 +426,76 @@ func (s *IntegrationTestSuite) TestCLIQueryTxCmdByEvents() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestCLIQueryTxsCmdByEvents() {
+	val := s.network.Validators[0]
+
+	account2, err := val.ClientCtx.Keyring.Key("newAccount2")
+	s.Require().NoError(err)
+
+	sendTokens := sdk.NewInt64Coin(s.cfg.BondDenom, 10)
+
+	// Send coins.
+	out, err := s.createBankMsg(
+		val, account2.GetAddress(),
+		sdk.NewCoins(sendTokens),
+	)
```

Comment:
Shouldn't we be explicitly setting the fee in this test case?

Reply:
I believe the test just queries the default test fee of `sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String())`